### DISCUSSION TITLE
CD images BlueSCSI v1.1

### DISCUSSION THREAD

**Discussion by trueskalon** [BlueSCSI/BlueSCSI-v2#213] [Help]
Hi, I got my hands on BlueSCSI v1.1 and its working for HD images but I got problems with CDs. BlueSCSI mounts them but they are not showing on desktop. Ive tried few isos and toasts without luck (from verifed sources). 

Can anyone give me some hints?

Thanks

**Comment by erichelgeson** [comment]
This is the v2 repo - head on over to the v1 Discussions here https://github.com/erichelgeson/BlueSCSI/discussions.

But to quickly answer your question - no there's no known issues with CD's using the latest firmware on v1. Note that not all ISO/toast images are ripped correctly and not all MacOS versions come with all drivers needed. 

If you still have questions please continue on the v1 discussions.
